docker-compose.yml

```
@@ -0,0 +1,14 @@
+services:
+  postgres-db:
+    container_name: postgres-db
+    image: postgres:15.10
+    platform: linux/x86_64
+    restart: always
+    environment:
+      - POSTGRES_USER=${POSTGRES_USER:-root}
+      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD:-rootpassword}
+      - POSTGRES_DB=${POSTGRES_NAME-habits:-todo}
```

Essa configuração de variáveis tá meio esquisita e ruim de entender, o hífen serve para substituir a variável de ambiente do contexto do env caso ela não exista ou esteja vazia, além disso, no yaml definimos `chave: valor` para melhor legibilidade.
```suggestion
      - POSTGRES_USER: ${POSTGRES_USER-root}
      - POSTGRES_PASSWORD: ${POSTGRES_PASSWORD-rootpassword}
      - POSTGRES_DB: ${POSTGRES_NAME-habits-todo}
```